index.html

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```

Comment:
You've done well creating the works and about sections but please note that you are not supposed to create a new repository for every new milestone😅. You are supposed to use the same repository and create  new branches for each milestone to be merged into the main branch. This way, you build out your portfolio in one repository step by step. Please add your approved code for the toolbar and headline sections into this repository and so your portfolio will have all sections🙏.